src/main/java/edu/kis/powp/jobs2d/drivers/DriverObserver.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverObserver implements Subscriber {
```

Comment:
The name of the class should also explain its purpose. (If not all observers would be called DriverObserver)